plugins/BEdita/API/tests/TestCase/Controller/ObjectsControllerTest.php

```diff
@@ -2863,6 +2863,7 @@ public function testParentsPermissions(): void
         $expected = [
             'roles' => ['second role'],
             'inherited' => false,
+            'descendant_have_perms' => false,
```

Comment:
Not sure that `descendant_have_perms` can have a straightforward meaning... It could be confused with the meaning "the descendant folders have permissions set". We want a label that tells "the authenticated user roles grants full access to descendant folders" 
Maybe we could use another label, as `descendant_perms_granted`.

Reply:
Completely agree, naming things is hard